skins/cat17/src/scripts/donationForm.js

```diff
@@ -333,18 +333,23 @@ $( function () {
 		initData.data( 'initial-validation-result' )
 	) );
 
-	var $introBanner = $('.introduction.banner');
-  var $introDefault = $('.introduction.default');
-
-  // @todo Check if this are all conditions that would be considered "successful deeplink", warrant the special header
-  if (initSetup.amount && initSetup.paymentIntervalInMonths && initSetup.paymentType) {
-    $introBanner.removeClass('hidden');
-    $introDefault.addClass('hidden');
-  }
-
 	// Non-state-changing event behavior
 
 	var scroller = WMDE.Scrolling.createAnimatedScroller( $( '.wrap-header, .state-bar' ) );
+
+	// Scroll to first required element that needs to be filled
+	var currentState = store.getState();
+	if ( currentState.validity.paymentData ) {
+		var nextRequired = currentState.donationFormContent.paymentType === 'BEZ' ? $( '#payment-method' ) : $( '#donation-type' );
+		var $introBanner = $('.introduction-banner');
+		$introBanner.insertBefore( nextRequired ).removeClass( 'hidden' );
+		// 24px = 1 em padding of intro-banner that is removed automatically by scroller, plus .5em whitespace above
+		var scrollOptions = { additionalOffset: -24 };
+		scroller.scrollTo( $introBanner, scrollOptions );
```

Comment:
Maybe move the `scrollOptions` object right into the .scrollTo? We still have the comment to explain it. OTOH, please see my comment for `scrolling.js`.